Client/UserResearch/OnboardingUserResearch.swift

```diff
@@ -7,25 +7,28 @@ import Leanplum
 import Shared
 
 struct LPVariables {
-    static var showOnboardingScreen = LPVar.define("showOnboardingScreen", with: true)
+    // Variable Used for AA test
+    static var showOnboardingScreenAA = LPVar.define("showOnboardingScreen", with: true)
+    // Variable Used for AB test
+    static var showOnboardingScreenAB = LPVar.define("showOnboardingScreen_2", with: true)
 }
 
 enum OnboardingScreenType: String {
-    case versionV1 // Default
-    case versionV2 // New version 2
+    case versionV1 // V1 (Default)
+    case versionV2 // V2
```

Comment:
```
static func from(boolValue: Bool) {
...
}

Reply:
Making it private(set) won't allow HiddenSettings to clear AB variables. 
I can put a specific method to clear those values in this class if you'd like that. What do you think?